openml-lightgbm-meta-module/openml-lightgbm/pom.xml

```diff
@@ -37,8 +37,8 @@
         <dependency>
             <!--add .lightgbm.-->
             <groupId>com.feedzai.openml.lightgbm.lightgbmlib</groupId>
-            <artifactId>lightgbmlib</artifactId>
             <version>${lightgbmlib.version}</version>
+            <artifactId>lightgbmlib</artifactId>
```

Comment:
The usual order is `groupId` -> `artifactId` -> `version` 😅

Reply:
> The usual order is `groupId` -> `artifactId` -> `version` sweat_smile

Interesting... I didn't notice this one :fearful: